deploy/azure/app/servicebus/outputs.tf

```diff
@@ -33,18 +33,18 @@ output "servicebus_subscription_filtered_id" {
   value       = azurerm_servicebus_subscription.sb_sub_2.id
 }
 
-output "function_publisher_id" {
-  value = azurerm_function_app.function_publisher.id
-}
+# output "function_publisher_id" {
```

Comment:
Is this no longer required or being commented out until a later story?

Reply:
These are outputs from the functions so they need to be commented out due to the commented out code above